address.go

```diff
@@ -174,12 +251,22 @@ type AddressPubKeyHash struct {
 	prefix string
 }
 
-// NewAddressPubKeyHash returns a new AddressPubKeyHash.  pkHash mustbe 20
+// NewAddressPubKeyHash returns a new AddressPubKeyHash. pkHash must be 20
 // bytes.
 func NewAddressPubKeyHash(pkHash []byte, net *chaincfg.Params) (*AddressPubKeyHash, error) {
 	return newAddressPubKeyHash(pkHash, net)
 }
 
+// NewSlpAddressPubKeyHash returns a new SLP AddressPubKeyHash.
+func NewSlpAddressPubKeyHash(pkHash []byte, net *chaincfg.Params) (*AddressPubKeyHash, error) {
```

Comment:
Is this better since it makes sure to preserve the original method's behavior?  If so, I will apply in-kind to the other similar wrappers.

```
func NewSlpAddressPubKeyHash(pkHash []byte, net *chaincfg.Params) (*AddressPubKeyHash, error) {
	addr, err := newAddressPubKeyHash(pkHash, net)
	if addr != nil {
		addr.prefix = net.SlpAddressPrefix
	}
	return addr, err
}
```

Reply:
I went ahead and pulled the trigger on this one, make sense and also improved coverage.  I also just noticed in vscode after running tests the go plugin provides coverage highlighting, this is very cool!

![image](https://user-images.githubusercontent.com/988195/103099537-dc88b300-45dc-11eb-95a7-257893ce7b13.png)